src/app/Dashboard Contents/header/header.component.html

```diff
@@ -18,19 +18,20 @@
           <a class="nav-link active" aria-current="page" href="#">Home</a>
         </li>
         <li class="nav-item">
-          <a class="nav-link" href="#">Link</a>
+          <a class="nav-link" href="#">Forum</a>
         </li>
         <li class="nav-item">
-          <a class="nav-link disabled"
-            >Disabled</a
-          >
+          <a class="nav-link" href="#">Medicine</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="..\..\Login Screen Components\login\login.component.html" >Login/Sign up</a>
```

Comment:
Use Routing and RouterLink to navigate to pages. Otherwise you're leaving the context of a single page application.
https://angular.io/api/router/RouterLink